.vim/coc-settings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "rust-analyzer.linkedProjects": [
+    "Cargo.toml",
+    "benchmarks/Cargo.toml",
+    "examples/Cargo.toml",
+    "risc0/zkvm/methods/guest/Cargo.toml",
+    "risc0/zkvm/methods/std/Cargo.toml",
```

Comment:
does this run the rust analyzer on guest code as well as the host side?

Reply:
I'm actually not sure. I copied these lines from the `.vscode/settings.json`
https://github.com/risc0/risc0/blob/main/.vscode/settings.json#L71-L77

Reply:
Oh right, (that's the editor I use, haha). For vscode rust analyzer is running on the guest code as seen by type annotations and code completion suggestions